src/V3EmitCMake.cpp

```diff
@@ -88,7 +84,8 @@ class CMakeEmitter final {
         *of << "\n### Constants...\n";
         cmake_set(*of, "PERL", V3Options::getenvPERL(), "FILEPATH",
                   "Perl executable (from $PERL)");
-        cmake_set(*of, "VERILATOR_ROOT", V3Options::getenvVERILATOR_ROOT(), "PATH",
+        cmake_set(*of, "VERILATOR_ROOT",
+                  V3OutFormatter::quoteNameControls(V3Options::getenvVERILATOR_ROOT()), "PATH",
```

Comment:
Ok, but then also please fix PERL immediately above, and any others.

Reply:
done